base/event/FairMCEventHeader.h

```diff
@@ -76,18 +85,18 @@ class FairMCEventHeader : public TNamed
     virtual void Register();
 
   protected:
-    UInt_t fRunId;      ///  Run identifier
-    UInt_t fEventId;    ///  Event identifier
-    Double32_t fX;      ///  Primary vertex x [cm]
-    Double32_t fY;      ///  Primary vertex y [cm]
-    Double32_t fZ;      ///  Primary vertex z [cm]
-    Double32_t fT;      ///  Event time [s]
-    Double32_t fB;      ///  Impact parameter [fm] (if relevant)
-    Int_t fNPrim;       ///  Number of input tracks
-    Bool_t fIsSet;      ///  Flag whether variables are filled
-    Double32_t fRotX;   ///  Rotation around x-axis (beam tilt) [rad]
-    Double32_t fRotY;   ///  Rotation around y-axis (beam tilt) [rad]
-    Double32_t fRotZ;   ///  Rotation around z-axis (event plane) [rad]
+    UInt_t fRunId;                  ///  Run identifier
+    EventID fEventId = NoEventID;   ///  Event identifier
```

Comment:
We once had a small discussion with @fuhlig1 that our _default_ (yes, I meant those) member initializers should have a consistent style using brances.

```suggestion
    EventID fEventId{NoEventID};   ///  Event identifier
```

I wonder, whether there is a clang-tidy / format option to actually enforce this. Didn't find one quickly.

Reply:
done